bzzeth/wire.go

```diff
@@ -25,6 +25,7 @@ var Spec = &protocols.Spec{
 	MaxMsgSize: 10 * 1024 * 1024,
 	Messages: []interface{}{
 		Handshake{},
+		DummyMessage{},
```

Comment:
I suppose that this message is here for testing until the protocol is finalized?

Reply:
yes